db/migrate/20160604175804_create_units.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUnits < ActiveRecord::Migration
+  def change
+    create_table :units do |t|
+      t.integer :building_id
+      t.string :name
+      t.integer :floor
```

Comment:
The `floor` field is gonna be a pain to maintain. People will be confused about whether floors start at 0 or 1 and how to handle sublevels and mezzanines. I think we should have it though. Some energy efficiency group just approached us about using our data and they wanted to know if we knew the floor of each apartment.